server/lib/service/dv360.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+import 'package:fixnum/fixnum.dart';
+import 'package:googleapis/displayvideo/v1.dart';
+import 'package:http/http.dart';
+
+/// A class that wraps around Display & Video 360.
+class DisplayVideo360 {
```

Comment:
As I mixed up the wrapper class and DV360 API server, would rename this class to something like DisplayVideo360Wrapper or DisplayVideo360Client help?

Reply:
Yes, that makes sense to me 👍